### PR TITLE
#define  TILES    1  //enable TILES.

### PR DESCRIPTION
**Pull request by chkngit**
hevc tiles

### CODE REVIEW

File: Source/API/EbApi.h [hunk 1]
```diff
@@ -151,7 +151,10 @@ extern "C" {
         // Quantization
         uint32_t                qp;
         uint8_t                 useQpFile;
-
+#if 1//TILES
```
Comment (tianjunwork) [inline Source/API/EbApi.h]: you meant "#if TILES"?

Comment (chkngit) [reply Source/API/EbApi.h]: yes, #define TILES , is defined in definitions.h , and it is not visible in the app, so I left it ON, but commented out to be able to track all changes when searching for TILES

**Comment by tianjunwork** [comment]
@chkngit Please fix the conflict and keep the original coding style.

File: Source/API/EbApi.h (hunk 1 above)
Comment (tianjunwork) [reply Source/API/EbApi.h]: "#if 1//TILES" is not necessary and confusing. "#if TILES" not necessary either. A internal parameter can be defined to enable/disable tile based on TileRowCount/TileColumnCount value.
e.g. if both TileRowCount/TileColumnCount > 1, tile is enabled.

**Pull request merged**